src/v/cluster/cluster_utils.h

```diff
@@ -229,6 +229,10 @@ ss::future<std::error_code> replicate_and_wait(
        &as = as,
        timeout,
        use_serde_serialization](controller_stm& stm) mutable {
+          if (!stm.throttle(cmd)) {
+              return ss::make_ready_future<std::error_code>(
+                errc::throttling_quota_exceeded);
+          }
```

Comment:
Should we log something here or does the Kafka layer do it on error?

Reply:
On error user will see appropriate error message.
We will see this error inside redpanda log in response error.